src/tomlToJSON.cc

```diff
@@ -21,6 +21,29 @@ namespace flox {
 
 /* -------------------------------------------------------------------------- */
 
+/** @brief An exception thrown when converting TOML to JSON */
+struct TOMLToJSONException : public FloxException {
+  static constexpr std::string_view categoryMsg = "error converting TOML to JSON";
+
+  std::filesystem::path dbPath;
```

Comment:
Why does the exception need to store the path to a database?

Reply:
I shouldn't have stored those paths in the pkgdb exception in retrospect. They didn't provide much value and were less ergonomic to use.

Its an example for how to stash context, but I didn't use it use it as sparingly aa I should have.

Reply:
Copy paste error 🙈

Reply:
Fixing